producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -70,143 +76,137 @@ public class SdtEndpointPortTypeTest extends AbstractSdtUnitTestBase {
-                .andThrow(new RuntimeException("test"));
-        EasyMock.replay(mockCreateBulkRequestHandler);
+    void testSubmitBulkException() {
+        when(mockCreateBulkRequestHandler.submitBulk(any())).thenThrow(new RuntimeException("test"));
 
         try {
             portType.submitBulk(createBulkRequest());
-            fail("Runtime exception should have been thrown");
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
         } catch (final RuntimeException re) {
-            assertEquals("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");
         }
 
-        EasyMock.verify(mockCreateBulkRequestHandler);
+        verify(mockCreateBulkRequestHandler).submitBulk(any());
     }
 
     /**
      * Test bulk feedback method completes successfully.
      */
     @Test
-    public void testBulkFeedbackSuccess() {
-        EasyMock.expect(mockBulkRequestHandler.getBulkFeedback(EasyMock.anyObject(BulkFeedbackRequestType.class)))
-                .andReturn(createBulkFeedbackResponse());
-        EasyMock.replay(mockBulkRequestHandler);
+    void testBulkFeedbackSuccess() {
+        when(mockBulkRequestHandler.getBulkFeedback(any(BulkFeedbackRequestType.class)))
+                .thenReturn(createBulkFeedbackResponse());
 
         final BulkFeedbackResponseType response = portType.getBulkFeedback(createBulkFeedbackRequestType());
 
-        assertNotNull("Response expected", response);
-        EasyMock.verify(mockBulkRequestHandler);
+        assertNotNull(response, RESPONSE_EXPECTED);
+        verify(mockBulkRequestHandler).getBulkFeedback(any(BulkFeedbackRequestType.class));
     }
 
     /**
      * Test bulk feedback method handles exceptions successfully.
      */
     @Test
-    public void testBulkFeedbackException() {
-        EasyMock.expect(mockBulkRequestHandler.getBulkFeedback(EasyMock.anyObject(BulkFeedbackRequestType.class)))
-                .andThrow(new RuntimeException("test"));
-        EasyMock.replay(mockBulkRequestHandler);
+    void testBulkFeedbackException() {
+        when(mockBulkRequestHandler.getBulkFeedback(any(BulkFeedbackRequestType.class)))
+                .thenThrow(new RuntimeException("test"));
 
         try {
             portType.getBulkFeedback(createBulkFeedbackRequestType());
-            fail("Runtime exception should have been thrown");
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
         } catch (final RuntimeException re) {
-            assertEquals("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");

```

Comment:
Might as well use version of assertEquals() that doesn't have the message parameter.